cmake/protobuf-module.cmake.in

```diff
@@ -1,7 +1,4 @@
-if(PROTOBUF_SRC_ROOT_FOLDER)
-  message(AUTHOR_WARNING "Variable PROTOBUF_SRC_ROOT_FOLDER defined, but not"
-    " used in CONFIG mode")
-endif()
+# Functions
```

Comment:
Instead of essentially copy-pasting FindProtobuf.cmake, couldn't we define the necessary variables and then include it?